### ISSUE TITLE
simple_talker PKT_SZ not changing

### ISSUE THREAD

**Issue by varun-v-deshpande** [AVnu/OpenAvnu#557]
Hello,

In simple talker application, I tried to change PKT_SZ to 200 but on wireshark at receiving end I still see only 98 Bytes.

Is it normal to happen ?

**Comment by pinealservo** [comment]
Changing PKT_SZ should change the length of the frames. Are you sure the code change got into the binary you are running?

**Comment by varun-v-deshpande** [comment]
Hello,

Yes, I am sure of that.
However, I would like to point few observations which I made.
Increasing PKT_SZ does increase the reserved bandwidth. I checked that on switch.

On the receiving side, When I checked with wireshark,  no matter what is my packet size, IPG, I always see AVB Transportation Protocol Packet 1722 of 98 Bytes.

On further observation I found that this 98 is nothing but 
18 + sizeof(*l4_headers) + (L4_SAMPLES_PER_FRAME * CHANNELS * L4_SAMPLE_SIZE );
